### DISCUSSION TITLE
How to use command line

### DISCUSSION THREAD

**Discussion by wisemonkey** [filips123/PWAsForFirefox#148] [Q&A] [answered]
I'm trying to use FirefoxPWAs on pinephone, since UI doesn't fit screen I wanted to use command line to install apps.
I see command to install site is:
`firefoxpwa site install MANIFEST_URL --profile PROFILE_ID`
However what is MANIFEST-URL in this case?

**Comment by filips123** [comment, marked as answer]
It is different for every website. I think the easiest and most universal way to find it is to open a website on a computer (or some device with a comfortably large screen), open developer tools (press F12), go to the Application tab, click on Manifest and see the URL.
